lib/oneview-sdk/resource/api200/volume.rb

```diff
@@ -68,11 +68,7 @@ def set_storage_system(storage_system)
       # @param [OneviewSDK::StoragePool] storage_pool Storage pool
       def set_storage_pool(storage_pool)
         assure_uri(storage_pool)
-        if self['provisioningParameters']
-          self['provisioningParameters']['storagePoolUri'] = storage_pool['uri']
-        else
-          self['storagePoolUri'] = storage_pool['uri']
```

Comment:
You've ommited this part in the new code. There is a reason why?

Reply:
Because Ricardo commented that it is not possible to update a storage pool, I checked this in the documentation. Do you think the previous form is better?

Reply:
You are saying you cannot update by changing the Storage Pool associated with the volume, right?

Reply:
Yes! This property is read only.